packages/itwinui-react/src/core/utils/components/Popover.tsx

```diff
@@ -64,8 +64,7 @@ export const Popover = React.forwardRef((props: PopoverProps, ref) => {
   const computedProps: Partial<TippyProps> = {
     allowHTML: true,
     animation: false,
-    appendTo: (el) =>
-      themeInfo?.portalContainerRef?.current || el.ownerDocument.body,
+    appendTo: (el) => themeInfo?.portalContainer || el.ownerDocument.body,
```

Comment:
I'm just wondering, is using `||` instead of `??` here and in all other places a personal preference or is there some benefit?

Reply:
it shouldn't matter in most cases, except `??` is for when you specifically only want to guard against `undefined` and `null`, whereas `||` will also consider booleans and other falsy values like empty strings